giraffe/src/types/index.ts

```diff
@@ -329,19 +329,6 @@ export interface TableGraphLayerConfig {
   properties: TableViewProperties
 }
 
-export interface FluxTable {
```

Comment:
This is a duplicate. The other one can be found here:

https://github.com/influxdata/giraffe/pull/285/files#diff-621c941aaa3ddff5f1fdb8deeca21291L427-L438